packages/connector-aliyun-dm/package.json

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "@logto/connector-aliyun-dm",
+    "version": "0.1.0",
+    "description": "Aliyun DM connector implementation.",
+    "main": "lib/index.js",
```

Comment:
better contains `exports` field due to compatibility

See Slack thread [discussion](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1644473115488029)